src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,10 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}>
+    <h2 className="TodoInfo__title">
+      {todo.title}
+    </h2>
+    <UserInfo user={todo.user} />
```

Comment:
The 'UserInfo' component is rendered even when the 'user' property of 'todo' might be undefined. It's a good practice to conditionally render components based on the availability of required data.